src/index.ts

```diff
@@ -317,7 +317,10 @@ type PipedDoWork<TLeft, TRight> = [TLeft] extends [HasDoWork<any, any>]
 // https://github.com/Microsoft/TypeScript/issues/29368#issuecomment-453529532
 type PipedRespond<TLeft, TRight> = [TLeft] extends [HasRespond<any, any>]
   ? [TRight] extends [HasRespond<any, any>]
-    ? HasRespond<RespondContextIn<TLeft>, RespondContextOut<TRight>>
+    ? HasRespond<
```

Comment:
this is needed because if we don't do this we will have a problem with respond function return type
if we have 2 parameters in HTPipe and they both have respond function the return type of piped respond function will be something like this:
{unsafeResponse: {unsafeResponse: unsafeResponseBody}}
if we will have 3 parameters in HTPipe and they all have respond function the return type of piped respond function will be something like this:
{unsafeResponse: {unsafeResponse: {unsafeResponse: unsafeResponseBody} } }

Reply:
good catch - but maybe we should define RespondContextOut differently instead?  Where else is it used?
- [ ] consistency?

Reply:
yup you right we can change it there
changed in new PR: https://github.com/captaincaius/hipthrusts/pull/78/files